src/onedrivesdk/model/async_operation_status.py

```diff
@@ -29,8 +29,8 @@
 
 class AsyncOperationStatus(OneDriveObjectBase):
 
-    def __init__(self, prop_dict={}):
-        self._prop_dict = prop_dict
+    def __init__(self, prop_dict=None):
+        self._prop_dict = prop_dict if prop_dict else {}
```

Comment:
> self._prop_dict = prop_dict if prop_dict else {}

I would go with `prop_dict if prop_dict is not None else {}` because it reads a bit better